.github/workflows/crowdin-download.content.yml

```diff
@@ -0,0 +1,391 @@
+name: i18n - Download Content
+on:
+  workflow_dispatch:
+  schedule:
+    # runs Monday and Wednesday at 12:15 PM UTC
+    - cron: '15 12 * * 1,3'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+  CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
+  CROWDIN_API_URL: 'https://freecodecamp.crowdin.com/api/v2/'
+  CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_content }}
+
+jobs:
+  i18n-download-content-translations:
+    name: Content
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        node-version: [20.x]
+      fail-fast: true
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          token: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+
+      - name: Generate Crowdin Config
+        uses: freecodecamp/crowdin-action@main
+        env:
+          PLUGIN: 'generate-config'
+          PROJECT_NAME: 'content'
```

Comment:
This will need to be created and set up in the crowdin action first.

Reply:
I apologize for my confusion, but how do we do this? I'm not familiar with where the Crowdin documentation lives.

Reply:
There's not really any documentation. What I need is:

- Where the english files are located
- Where we want the i18n versions to go

Reply:
Okay so the English markdown files are in  "src/content/docs" at the root. The foreign language version of the markdown docs live in each of the language folders (except images; I really should move those soon). 

As for the i18n files (or what was intended to be those), they live in "src/i18n" and in this PR of the JSON file type.